docs/views/spec/other.mdx

```diff
@@ -9,10 +9,10 @@ The following view elements can be used in your Zeno Projects.
 
 A plaintext view to display raw text data. See in [this example](https://zeno-ml.github.io/zeno-instance-views/?params=eyJzYW1wbGUiOiJ0ZXh0LWNsYXNzaWZpY2F0aW9uIn0=).
 
-<CodeBlock language="json" title="Separated Values">
+<CodeBlock language="typescript" title="Separated Values">
```

Comment:
should probably keep these unique. Since people are using this in python I used json instead of TS.